src/main/java/de/mediathekview/mlib/tool/FileSize.java

```diff
@@ -1,11 +1,11 @@
-package mSearch.tool;
+package de.mediathekview.mlib.tool;
+
+import java.io.IOException;
 
 import okhttp3.Request;
 import okhttp3.Response;
 import okhttp3.ResponseBody;
 
-import java.io.IOException;
-
 public class FileSize {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)